packages/rich-text/src/to-tree.js

```diff
@@ -145,15 +146,14 @@ export function toTree( {
 					return;
 				}
 
-				const { type, attributes, object } = format;
 				const parent = getParent( pointer );
-				const newNode = append( parent, fromFormat( { type, attributes, object } ) );
+				const newNode = append( parent, fromFormat( format ) );
```

Comment:
Do we no longer need to clone it? Just making sure it was done on purpose.

Reply:
No, the cloning is not needed. I think it's a remnant, also to destruct `object`.